apps/dashboard/components/logs/queries/queries-overflow-tooltip.tsx

```
@@ -0,0 +1,29 @@
+  return (
+    <Tooltip>
+      <TooltipTrigger>
+        <QueriesPill value={`+${list.length} more`} className="text-gray-10" />
+      </TooltipTrigger>
+      <TooltipContent
+        className="flex h-full bg-white text-gray-12 rounded-lg font-500 text-[12px] justify-center items-center leading-6 shadow-lg"
+        side="bottom"
+      >
+        <ul className="flex flex-col gap-2 p-2">
+          {list?.map((item) => (
+            <QueriesPill value={item.value} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)